mappings/net/minecraft/entity/data/TrackedDataHandler.mapping

```diff
@@ -1,9 +1,28 @@
 CLASS net/minecraft/unmapped/C_xrzqxzbg net/minecraft/entity/data/TrackedDataHandler
+	METHOD m_dybaoeou (Lnet/minecraft/unmapped/C_feaetnaz;Lnet/minecraft/unmapped/C_idfydwco;)Ljava/lang/Object;
+		ARG 1 packetByteBuf
 	METHOD m_hkkrjyzt create (I)Lnet/minecraft/unmapped/C_rinmcaxy;
+		ARG 1 id
+	METHOD m_hvvazhmx createOptional (Ljava/util/function/BiConsumer;Ljava/util/function/Function;)Lnet/minecraft/unmapped/C_xrzqxzbg;
+		ARG 0 serializer
+		ARG 1 deserializer
 	METHOD m_hzlqlpnz copy (Ljava/lang/Object;)Ljava/lang/Object;
 		ARG 1 value
+	METHOD m_idvptrbv (Ljava/lang/Class;Lnet/minecraft/unmapped/C_idfydwco;)Ljava/lang/Enum;
+		ARG 1 packetByteBuf
 	METHOD m_maxtjmhy write (Lnet/minecraft/unmapped/C_idfydwco;Ljava/lang/Object;)V
 		ARG 1 buf
 		ARG 2 value
+	METHOD m_nbfbpgbm (Lnet/minecraft/unmapped/C_feaetnaz;Lnet/minecraft/unmapped/C_idfydwco;Ljava/lang/Object;)V
+		ARG 1 packetByteBuf
+		ARG 2 o
+	METHOD m_oiewqxlp createIndexed (Lnet/minecraft/unmapped/C_feaetnaz;)Lnet/minecraft/unmapped/C_xrzqxzbg;
+		ARG 0 iterable
+	METHOD m_paobprsl createEnum (Ljava/lang/Class;)Lnet/minecraft/unmapped/C_xrzqxzbg;
+		ARG 0 clazz
+	METHOD m_slncelbp create (Ljava/util/function/BiConsumer;Ljava/util/function/Function;)Lnet/minecraft/unmapped/C_xrzqxzbg;
+		ARG 0 serializer
+		ARG 1 deserializer
 	METHOD m_tbdwkthw read (Lnet/minecraft/unmapped/C_idfydwco;)Ljava/lang/Object;
 		ARG 1 buf
+	CLASS C_knjazpcj SimpleTrackedDataHandler
```

Comment:
`TrackedDataHandler$SimpleTrackedDataHandler` seems redundant, no?

Reply:
I think it extends it right its a class

Reply:
`SimpleHandler` or `SimpleDataHandler` would work